hex/Hex2.cs

```diff
@@ -242,7 +257,7 @@ class Hex2 : UserControl {
 
                 rect.Width = charWidth;
                 for (int i = 0; i < numColumns; i++) {
-                    var num = ((i % 15) + 1);
+                    var num = ((i % 16));
```

Comment:
Ok that's fine. Most hex editors I've seen have 16 columns so that the data will align better with the actual memory address. 
![hex](https://cloud.githubusercontent.com/assets/12224394/8436086/b317462a-1f23-11e5-8549-c1a7a5e32721.png)

If you don't mind me asking, why do you prefer it start at 1?